src/index.html

```diff
@@ -1,22 +1,333 @@
+<!-- #region HEAD -->
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+    <title>Bang & Olufsen</title>
+    <link
+    rel="icon"
+    href="../src/images/Logo-phone-320.png"
+    type="image/png"
+    />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="./styles/style.scss"
```

Comment:
utilizar o main scss

Reply:
fiz as mudanças mas agora quando vou fazer o deploy surge a seguinte mensagem: ➜  layout_landing-page git:(develop) ✗ npm run deploy                               

> layout_landing-page@1.0.0 deploy
> mate-scripts deploy

Start deploy to gh-pages. Please wait, it may take up to minute.

Warning: No html_report
Deploy error:  Command failed: git add /Users/lucas/projetos/layout_landing-page/dist -f

Reply:
ok. acredito que consegui resolver o problema acima mencionado. aguardo sua aprovação.